packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -222,9 +222,15 @@ const getAllProps = (
     userSelect: getResponsiveValue(props.userSelect, breakpoint),
     pointerEvents: getResponsiveValue(props.pointerEvents),
     opacity: getResponsiveValue(props.opacity, breakpoint),
+    visibility: getResponsiveValue(props.visibility, breakpoint),
     ...(!isReactNative() && {
       boxShadow: getElevationValue(props.elevation, props.theme, breakpoint),
     }),
+
+    // Polygon support
+    transform: getResponsiveValue(props.transform as string, breakpoint),
```

Comment:
do we need this now since we are approaching rebranding?

Reply:
Does re-branding ensure no more polygons?

Reply:
Talked to Chaitanya about this today. There are going to be polygons after rebranding too but they will be built on frontend-website repo.

I think we still might have to expose this to enable them to build Polygons using Box component (since that covers major part of homepage)